WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/download/BackupDownloadActivity.kt

```diff
@@ -102,6 +111,22 @@ class BackupDownloadActivity : LocaleAwareActivity() {
         viewModel.start(savedInstanceState)
     }
 
+    private fun SnackbarMessageHolder.showSnackbar() {
+        val snackbar = WPSnackbar.make(
+                coordinator_layout,
+                uiHelpers.getTextOfUiString(this@BackupDownloadActivity, this.message),
+                Snackbar.LENGTH_LONG
+        )
+        if (this.buttonTitle != null) {
+            snackbar.setAction(
+                    uiHelpers.getTextOfUiString(this@BackupDownloadActivity, this.buttonTitle)
+            ) {
+                this.buttonAction.invoke()
```

Comment:
Np: `this` can be omitted before `message`, `buttonTitle`, `buttonAction`

Reply:
Updated . Thanks